components/cam/bld/namelist_files/use_cases/2010_scream_hr.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0"?>
+<namelist_defaults>
+
+<!-- TODO: enable COSP once we do an E3SM->SCREAM merge -->
+
+<!-- Solar constant from CMIP6 input4MIPS -->
+<solar_data_file>atm/cam/solar/Solar_2010control_input4MIPS_c20181017.nc</solar_data_file>
+<solar_data_ymd>20100101</solar_data_ymd>
+<solar_data_type>FIXED</solar_data_type>
+
+<!-- 2010 GHG values from CMIP6 input4MIPS -->
+<!-- <co2vmr>312.821e-6</co2vmr> The CMIP6 concentration set by CCSM_CO2_PPMV in
+     cime/src/drivers/mct/cime_config/config_component_acme.xml -->
+<ch4vmr>1807.851e-9</ch4vmr>
+<n2ovmr>323.141e-9</n2ovmr>
+<f11vmr>768.7644e-12</f11vmr>
+<f12vmr>531.2820e-12</f12vmr>
+
+<!-- Stratospheric aerosols from CMIP6 input4MIPS -->
+<prescribed_volcaero_datapath>atm/cam/volc</prescribed_volcaero_datapath>
+<prescribed_volcaero_file>    CMIP_DOE-ACME_radiation_average_1850-2014_v3_c20171204.nc</prescribed_volcaero_file>
+<prescribed_volcaero_filetype>VOLC_CMIP6  </prescribed_volcaero_filetype>
+<prescribed_volcaero_type>    CYCLICAL    </prescribed_volcaero_type>
+<prescribed_volcaero_cycle_yr>1           </prescribed_volcaero_cycle_yr>
+
+<!-- Ice nucleation mods-->
+<use_hetfrz_classnuc    >.false.</use_hetfrz_classnuc>
```

Comment:
I'm not sure we want to turn off classical nucleation theory. This was a hack in E3SMv1 HR to compensate for bergeron efficiency being scaled by a ridiculous factor of 0.1 in v1. I'm not totally sure it matters for prescribed aero, but I suspect it does. I am, however, not sure whether turning classical nucleation theory back on and fixing the bergeron factor would require complete retuning. So consider this a bookmark for 'more investigation needed'.

Reply:
Update: I asked CHris Golaz his opinion on using classical nucleation. Also, I pondered whether we're even using this scheme if we have prescribed aerosol. I think the answer is yes - this proc takes in an aerosol distribution and computes the ice nuclei it should have, so it will work any time we don't specify ice number.

Reply:
Update^2: Chris pointed out that our parametric complexity for heterogeneous nucleation should match that of the rest of our parameterizations. This means we should definitely use Meyers, which doesn't depend on aerosols which we're not modeling anyways. So .false. is definitely the right answer. But this makes me realize that we'll eventually need to port Meyers scheme to C++...